.github/workflows/pull.yml

```diff
@@ -215,6 +215,11 @@ jobs:
           { config: "default", shard: 1, num_shards: 3, runner: "linux.2xlarge" },
           { config: "default", shard: 2, num_shards: 3, runner: "linux.2xlarge" },
           { config: "default", shard: 3, num_shards: 3, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
```

Comment:
Do we really need crossref tests on 3.12 vs 3.11?

Reply:
Removed for now